### ISSUE TITLE
Let cpp extension tooltip support \code tag in comment.

### ISSUE THREAD

**Issue by habaohaba** [microsoft/vscode-cpptools#12114]
### Feature Request

Currently, I don't find the configuration to let cpp tooltip correctly render \code section in comment, so I request this feature which I think is really helpful.

**Comment by browntarik** [comment]
Could you clarify the behavior of this tag? Are you referring to a Doxygen Tag?

**Comment by sean-mcmanus** [comment]
Yeah, they're referring to https://www.doxygen.nl/manual/commands.html#cmdcode .

We currently support markdown "code" formatting via 3 backticks.
